stripe/src/main/java/com/stripe/android/view/PaymentMethodsActivity.kt

```diff
@@ -180,7 +179,7 @@ class PaymentMethodsActivity : AppCompatActivity() {
         viewModel.getPaymentMethods().observe(this, Observer {
             when (it) {
                 is PaymentMethodsViewModel.Result.Success -> {
-                    adapter.setPaymentMethods(it.paymentMethods)
+                    adapter.setPaymentMethods(it.paymentMethods, viewModel.selectedPaymentMethodId)
```

Comment:
Instead of doing this here and changing `PaymentMethodsAdapter#setPaymentMethods()`, I would update `onAddedPaymentMethod()` to set `PaymentMethodsAdapter#selectedPaymentMethodId` to the id of the newly added payment method.

Reply:
oh I thought since it was internal I wouldn't be able to do that 👍